cfme/tests/infrastructure/test_host.py

```diff
@@ -560,3 +561,48 @@ def test_infrastructure_hosts_crud(appliance, setup_provider_min_hosts, provider
     except UnexpectedAlertPresentException:
         pytest.fail("Abandon changes alert displayed, but no changes made.")
     # TODO add additional crud functionality.
+
+
+@test_requirements.infra_hosts
```

Comment:
Can this be moved to the `pytestmark` at the top of the test file and removed on each individual test case? Or are there test cases that